repos/system_upgrade/el8toel9/actors/nischeck/tests/test_nischeck.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+
+from leapp import reporting
+from leapp.libraries.actor import nischeck
+from leapp.libraries.common.testutils import create_report_mocked, CurrentActorMocked
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM, NISConfig, RPM
+
+
+def _generate_rpm(name, version, release, epoch):
+    """
+    Generate new RPM model.
+
+    Parameters:
+        name (str): rpm name
+        version (str): rpm version
+        release (str): rpm release
+        epoch (str): rpm epoch
+
+    Returns:
+        rpm  (RPM): new RPM object with name parameter set
+    """
+    return RPM(name=name,
+               version=version,
+               release=release,
+               epoch=epoch,
+               pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID 199e2f91fd431d51',
+               packager='Red Hat, Inc. <http://bugzilla.redhat.com/bugzilla>',
+               arch='noarch')
```

Comment:
A small nitpick: This function can be replaced by `functools.partial`:
```suggestion
import functools
_generate_rpm = functools.partial(RPM,
                                  pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID 199e2f91fd431d51',
                                  packager='Red Hat, Inc. <http://bugzilla.redhat.com/bugzilla>',
                                  arch='noarch')
```
And then used as follows:
```
_generate_rpm(name='<name>', version='<version>', release='<release>', epoch='<epoch>')
```
Using keyword arguments helps with readability.

Reply:
Changed.